exchange/price_granularity.go

```diff
@@ -15,21 +16,27 @@ func GetCpmStringValue(cpm float64, config openrtb_ext.PriceGranularity) (string
 	bucketMax := 0.0
 	increment := 0.0
 	precision := config.Precision
-	// calculate max of highest bucket
+
+	// Limit the number of decimal significant figures. Very large values lead to Panics
+	if precision > 4 {
```

Comment:
This needs review from @camrice and/or @VeronikaSolovei9. Can you please explain how large values lead to panics?

Reply:
I tested current upstream implementation with precision = 10 and it works. 
The only thing it looks scary in response: 
```
{
                    "hb_pb": "17.0000000000",
                    "hb_pb_cat_dur": "17.0000000000_food_30s",
                    "hb_cache_id": "37be5f83-4be9-41b4-ab3f-84883205591d"
                }
```

Reply:
@VeronikaSolovei9 do you think we should should put a limit to `precision`? If so, what should it be?

Reply:
Maybe we should not return error in this case, instead just overwrite precision with maximum reasonable value?

Reply:
@guscarreon 2 and 4 sounds reasonable to me, but please confirm it with product owners as well.

Reply:
In this case, the product owners are the Prebid Server Committee. This is already configurable by the publisher. What's the worst thing that would happen if they enter a value too large? Is this really something we need to code logic for?

Reply:
"hb_pb_cat_dur" is a targeting key for Ad server to select Line Items. Redirect URL also contains it. With wrong precision Ad server set up will not work.